src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private int CONSTANT_CASE = 100;
```

Comment:
```suggestion
    private static final int CONSTANT_CASE = 100;
```
Please, rename this variable
using CONSTANT_CASE means that all constants should be in upper case and use underscore
but the variable name should give information about what it is used for
(ex `MAX_BALL_NUMBER`)

Reply:
oh, sorry. it was a hard day.